Examples/elemental.cattle.io/v1beta1/MachineRegistration.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: elemental.cattle.io/v1beta1
+kind: MachineRegistration
+metadata:
+  name: test-machine
+spec:
+  config:
+    data:
+      write_files:
+        content: V2h5IGFyZSB5b3UgY2hlY2tpbmcgdGhpcz8K
+        encoding: b64
+    elemental:
+      install:
+        device: /dev/vda
+        iso: https://something.example.com
```

Comment:
I understand the registration and system agent data is of the `config.Config.Elemental` is irrelevant here, right?

Reply:
yes, registration data at least should be generated by the server (ca-cert, registration-url, token)

The system agent data I have no idea because Is till don't know what it does or even what format it should take...